sevntu-checks/src/main/resources/com/github/sevntu/checkstyle/checks/coding/messages.properties

```diff
@@ -19,6 +19,7 @@ finalize.implementation.public = finalize() method should have a "protected" vis
 finalize.implementation.useless = finalize() method is useless: it does nothing except for calling super.finalize().
 finalize.implementation.missed.super.finalize = You have to call super.finalize() right after finally opening brace.
 forbid.certain.imports=Import ''{1}'' should not match ''{0}'' pattern, it is forbidden.
+forbid.certain.methods=Call to ''{0}'' method (matches pattern {1}) with ''{2}'' arguments (matches pattern {3}) is forbidden.
```

Comment:
> with ''{2}'' arguments

This won't make much sense if argument is a regular expression.
Would it be better to split this between `argCount` set and `argCount` not set?

`Call to method ''{0}'' is forbidden.`
`Call to method ''{0}'' with number of arguments matching ''{1}'' is forbidden.`

This would make it clear if it is just method that is forbidden or if it is only certain number of arguments.

Reply:
I will make the change. Thanks for pointing out.

Reply:
Done.